test/rendering-test/webpack.config.test-rendering.js

```diff
@@ -25,16 +25,20 @@ module.exports = {
 
   resolve: {
     alias: {
-      'deck.gl': resolve('./src')
+      'deck.gl': resolve('../../src')
     }
   },
 
   module: {
     rules: [
       {
-        include: [resolve('./src')],
+        include: [resolve('../../src')],
         loader: 'transform-loader',
         options: 'brfs-babel'
+      },
+      {
+        test: /\.png$/,
```

Comment:
Maybe add some comment on what this loader does that can't be handled by brfs. Is it correct to assume that these test cases don't work directly under node.js, since they depend on a loader?